tests/dist.rs

```diff
@@ -496,24 +498,50 @@ fn uninstall(
     Ok(())
 }
 
+#[derive(Copy, Clone, Debug)]
+enum Compressions {
+    GZOnly,
+    AddXZ,
+    AddZStd,
+    AddBoth,
```

Comment:
AddBoth isn't very meaningful without the context of Compressions::

Reply:
Fair, changed to `AddXZAndZStd` since ultimately it's not actually used (yet)